libopae/plugins/ase/CMakeLists.txt

```diff
@@ -75,12 +75,17 @@ add_subdirectory(rtl)
 set(EXTRA_ASE_FILES
   Makefile
   ase.cfg
-  ase_regress.sh)
+  ase_regress.sh
+  opae_ase.cfg)
 
 install(FILES ${EXTRA_ASE_FILES}
   DESTINATION ${ASE_INST_SHARE_DIR}
   COMPONENT aseextra)
 
+install(FILES opae_ase.cfg
+  DESTINATION $ENV{HOME}/.local/opae
```

Comment:
can this file not be installed in the home directory? is there a specific need for this file to be in the home directory? it creates problems when specifying file paths in the rpm specfile. @tswhison what do you think?

Reply:
I have updated the rpm specfile to incorporate the new changes but I wasn't able to get the home directory to be identified by the specfile.

Reply:
Ooh, I can see how the home directory would be problematic for the rpm spec file.  It's probably better that it go into /etc/opae for that reason alone.

Reply:
It's installed in <install root>/share/opae/ase/ too. Is that sufficient and you can skip both .local and /etc?

Reply:
Sure, I will remove it from $HOME.